bin/start.ts

```diff
@@ -26,7 +28,7 @@ fork(function*() {
       testManifestPath: tempy.file({ name: 'manifest.js' }),
     }));
 
-    yield receive({ ready: "orchestrator" });
+    yield receive({ ready: "orchestrator" }, orchestrator);
```

Comment:
argument order of receive has been switched to follow the FP convention of most general arguments coming first, more specific arguments coming later.